src/site/layouts/va_form.drupal.liquid

```diff
@@ -68,9 +68,24 @@
           {{ fieldVaFormUsage.processed }}
 
           <h3>Downloadable PDF</h3>
-          <a href="{{ fieldVaFormUrl.uri }}" download>Download VA Form {{ fieldVaFormNumber }} (PDF)</a>
+          <a
+            href="{{ fieldVaFormUrl.uri }}"
+            target="_blank"
+            download
+            data-widget-type="find-va-forms-invalid-pdf-alert"
+            data-form-number="{{ fieldVaFormNumber }}">
+            Download VA Form {{ fieldVaFormNumber }} (PDF)
+          </a>
         {% else %}
-          <a class="vads-u-margin-top--4" href="{{ fieldVaFormUrl.uri }}" download>Download VA Form {{ fieldVaFormNumber }} (PDF)</a>
+          <a
+            class="vads-u-margin-top--4"
+            href="{{ fieldVaFormUrl.uri }}"
+            target="_blank"
+            download
+            data-widget-type="find-va-forms-invalid-pdf-alert"
+            data-form-number="{{ fieldVaFormNumber }}">
+            Download VA Form {{ fieldVaFormNumber }} (PDF)
+          </a>
```

Comment:
In building this, I attempted to use JS to just enhance the actual template with the invalid-PDF feature, so that the anchor tag itself is still part of the template.